.vscode/tasks.json

```diff
@@ -114,6 +114,19 @@
                 ]
             }
         },
+        {
+            "label": "Build Lighting App (Tizen)",
+            "type": "shell",
+            "command": "scripts/examples/gn_tizen_example.sh examples/lighting-app/linux out/lighting_app_tizen 'import(\"//with_tizen.gni\")'",
```

Comment:
why is the import required here? I believe other apps are not needing this.

Please update this to use scripts/build/build_examples.py

Reply:
I updated the command.